### ISSUE TITLE
Vuejs event attributes

### ISSUE THREAD

**Issue by akaspick** [slm-lang/slm#49]
I'm trying to use event attributes in vuejs which look like this...
```<a href="#" @click="doSomething()">Do Something</a>```

In slm
```a href="#" @click="doSomething()" Do Something``` doesn't work with the **@**

**Comment by Odaeus** [comment]
I write a lot of VueJS and haven't noticed a problem, a quick test on the NodeJS REPL seems to show it works as expected:

```
> var slm = require('slm');
> slm.render("button @click='test' Test")
'<button @click="test">Test</button>'
```

**Comment by akaspick** [comment]
@Odaeus Will try again and report back with my findings.  I'm doing this within a .vue component file which may be interfering somehow.

```
<template lang="slm">
a href="#" @click="doSomething()"
</template>
```

**Comment by akaspick** [comment]
@Odaeus Looks like I used single quotes instead of double quotes before the event attribute which caused the issue.

```a href="#" @click=""``` vs ```a href='' @click=""```

Apologies and thanks for verifying that it works.

**Comment by akaspick** [comment]
@Odaeus Spoke too soon... the issue appears when the event attribute has a modifier such as...
```a href="#" @click.stop="doSomething()"```

**Comment by Odaeus** [comment]
@akaspick yes the modifier breaks it, this was fixed in version 1.0.0. I hit the same problem just an hour ago. If you're using `slm-loader` like I am, it hasn't been updated and is locked to '0.5.x'. My current workaround is to add:
```
"resolutions": {
  "slm-loader/slm": "1.0.0"
}
```
to my `package.json` (I'm using Yarn if that makes a difference) to override it.

**Comment by akaspick** [comment]
@Odaeus Ah, perfect.  Thanks for the workaround.
